openhcl/virt_mshv_vtl/src/processor/mod.rs

```diff
@@ -675,7 +676,11 @@ impl<'p, T: Backing> Processor for UhProcessor<'p, T> {
                     }
                 }
 
-                if ready {
+                // TODO WHP GUEST VSM: This should be next_vtl.
+                // All other backings always return true for ready, as they either
```

Comment:
If that's the case, we should probably figure out how to move the wait into the mshv-specific backend code. But this is fine for now.

Reply:
https://github.com/microsoft/openvmm/pull/114 maybe better?